tests/dummy/app/templates/components/test-component.hbs

```diff
@@ -10,6 +10,12 @@
              after=object.text
              updated=(action "update")}}
 <br/>
+{{form-field type="date-picker"
+             object=object
+             propertyPath="date"
+             after=object.text
```

Comment:
Show the value of the field after it. The field will update `date`, so please show `date` after, not `text`